Source/Scene/Label.js

```diff
@@ -337,6 +353,82 @@ define([
         },
 
         /**
+         * Determines if a background behind this label will be shown.
+         * @memberof Label.prototype
+         * @type {Boolean}
+         */
+        showBackground : {
+            get : function() {
+                return this._showBackground;
+            },
+            set : function(value) {
+                //>>includeStart('debug', pragmas.debug);
+                if (!defined(value)) {
+                    throw new DeveloperError('value is required.');
+                }
+                //>>includeEnd('debug');
+
+                if (this._showBackground !== value) {
+                    this._showBackground = value;
+                    rebindAllGlyphs(this);
+                }
+            }
+        },
+
+        /**
+         * Gets or sets the background color of this label.
+         * @memberof Label.prototype
+         * @type {Color}
+         */
+        backgroundColor : {
+            get : function() {
+                return this._backgroundColor;
+            },
+            set : function(value) {
+                //>>includeStart('debug', pragmas.debug);
+                if (!defined(value)) {
+                    throw new DeveloperError('value is required.');
+                }
+                //>>includeEnd('debug');
+
+                var backgroundColor = this._backgroundColor;
+                if (!Color.equals(backgroundColor, value)) {
+                    Color.clone(value, backgroundColor);
+
+                    var backgroundBillboard = this._backgroundBillboard;
+                    if (defined(backgroundBillboard)) {
+                        backgroundBillboard.color = backgroundColor;
+                    }
+                }
+            }
+        },
+
+        /**
+         * Gets or sets the background padding, in pixels, of this label.  The <code>x</code> value
+         * controls horizontal padding, and the <code>y</code> value controls vertical padding.
+         * @memberof Label.prototype
+         * @type {Cartesian2}
+         */
+        backgroundPadding : {
+            get : function() {
+                return this._backgroundPadding;
+            },
+            set : function(value) {
+                //>>includeStart('debug', pragmas.debug);
+                if (!defined(value)) {
+                    throw new DeveloperError('value is required.');
+                }
+                //>>includeEnd('debug');
+
+                var backgroundPadding = this._backgroundPadding;
+                if (!Cartesian2.equals(backgroundPadding, value)) {
+                    Cartesian2.clone(value, backgroundPadding);
+                    repositionAllGlyphs(this);
```

Comment:
I believe [this delay already happens](https://github.com/AnalyticalGraphicsInc/cesium/blob/178d71253977472bcc1934705e26151e453da939/Source/Scene/Label.js#L36-L50).